crates/egui-winit/Cargo.toml

```
@@ -61,12 +61,12 @@ egui = { version = "0.24.0", path = "../egui", default-features = false, feature
 log = { version = "0.4", features = ["std"] }
 raw-window-handle.workspace = true
 web-time = { version = "0.2" } # We use web-time so we can (maybe) compile for web
-winit = { version = "0.28", default-features = false }
+winit = { version = "0.29", default-features = false, features = ["rwh_05"] }
```

Since `0.29.4` contains a critical fix, let's make sure all `egui-winit` users get it:

```suggestion
winit = { version = "0.29.4", default-features = false, features = ["rwh_05"] }
```